src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -554,4 +555,6 @@ void DlgPrefLibrary::slotSyncTrackMetadataToggled() {
     if (isVisible() && checkBox_SyncTrackMetadata->isChecked()) {
         mixxx::DlgTrackMetadataExport::showMessageBoxOncePerSession();
     }
+    // If Track Metatdata checkbox is unchecked, ignore the state of Serato Metatdata checkbox.
```

Comment:
It doesn't make sense to repeat and rephrase what the code does as a comment. And it is also inconsistent to insert a comment only here but not at the other place. Please remove it.